packages/augur-ui/src/modules/app/components/inner-nav/base-inner-nav-pure.tsx

```diff
@@ -50,9 +50,9 @@ const BaseInnerNavPure = () => {
   const sortProps = {
     setFilterSortState: updates => setFilterSortState({ ...filterSortState, ...updates }),
   };
-  const showMainMenu = mobileMenuState >= MOBILE_MENU_STATES.FIRSTMENU_OPEN;
+  const showMainMenu = mobileMenuState >= MOBILE_MENU_STATES.FIRSTMENU_OPEN && mobileMenuState <= MOBILE_MENU_STATES.SUBMENU_OPEN;
```

Comment:
I'm confused regarding the logic of "showMainMenu". When I tried to set a mobile menu state, it was opening what I wanted + filters [???] so I just limited that condition to the amount of options that we had and added a new mobileMenuState.